README.md

```diff
@@ -35,13 +85,31 @@ $ docker trust inspect raccos/releases-opensuse:cos-system-0.4.32
 
 ## Quick start
 
-Download the ISO from the latest [release]() and run it in your virtualization hypervisor of choice (baremetal, too). You can login with the user `root` and `cos`. That's a live ISO and no changes will be persisted.
+cOS releases consist on docker images that can be used to build derived against. cOS is a manifest which assembles an OS derived from docker images, so if you want to make substantial changes to the layout you can also fork directly cOS.
+
+The cOS CI generates ISO and images artifacts used for testing, so you can also try out cOS by downloading the 
+ISO from the Github Actions page, to the commit you are interested into.
```

Comment:
It would be nice to have a link to that GitHub Actions page. Minor thing...

Reply:
both fixed :+1: